### ISSUE TITLE
About dialog conversion detected issues

### ISSUE THREAD

**Issue by netonjm** [netonjm/FigmaSharp#224]
- [x] ImageResources https://github.com/netonjm/FigmaSharp/issues/225
with the current approach this generates in code generation 2 images 1 (hidden)
- [x] Fix accessibility AccessibilityTitles and AccessibilityHelps (not necessary in buttons) in layer design also missing or wrong titles
- [x] WantsLayer it's not necessary in some native controls like NSButton
- [x] Add a11 tile in AboutHero = new AppKit.NSImageView();
- [x] Rename SelectedContent to ActiveContent in https://github.com/netonjm/FigmaSharp/blob/master/FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/FigmaContainerBundleWindow.cs

**Comment by hbons** [comment]
WantsLayer removed in #234

**Comment by netonjm** [comment]
Fixed
